docs/releasing/publishing.md

```diff
@@ -48,32 +66,32 @@
   npm unlink ../govuk-frontend/package/
   ```
 
-11. Create a pull request and copy the changelog text.
+10. Create a pull request and copy the changelog text.
```

Comment:
Unclear where we're copying the changelog text to? I assume this means the PR description?

Reply:
I might leave this to be looked at later. We haven't touched this part of the doc apart from updating the numbering.